src/ios/OpenTokPlugin.m

```diff
@@ -403,7 +414,9 @@ - (void)subscribe:(CDVInvokedUrlCommand*)command{
     OTSubscriber* sub = [[OTSubscriber alloc] initWithStream:myStream delegate:self];
     sub.audioLevelDelegate = self;
     sub.networkStatsDelegate = self;
-    [_session subscribe:sub error:nil];
+    NSString* sessionId = [command.arguments objectAtIndex:10];
```

Comment:
move to the beginning of the function next to all the others `[command.arguments objectAtIndex`